spring-app/src/main/resources/application-localdb.properties

```diff
@@ -0,0 +1,25 @@
+spring.jpa.hibernate.ddl-auto=${UP_DDL_AUTO:validate}
+spring.datasource.driver-class-name=${UP_JDBC_DRIVER:org.mariadb.jdbc.Driver}
+server.port=${UP_PORT:8080}
+spring.servlet.multipart.enabled=false
+app.mariaDB4j.databaseName=${UP_DB_NAME_NAME:up_dev}
+mariaDB4j.port=3308
+mariaDB4j.dataDir=./database
+
+up.datastore=${UP_DATASTORE_PATH:uploads}
+
+#Maximum accepted payload in bytes.
+#Actual file size limits can be configured at runtime for each user, but cannot be larger than this value. Default 5GiB
+up.max-file-size=${UP_MAX_FILE_SIZE:5368709120}
+
+#in bytes. Default 4 * 1024 * 1024 (4MB)
+up.chunk-size=${UP_CHUNK_SIZE:4194304}
+
+#in milliseconds. Default 1h
+up.cleanup-interval=${UP_CLEANUP_INTERVAL:3600000}
+
+#Secret will be randomly generated at runtime if UP_JWT_SECRET variable is not set.
+#Warning: sessions will be invalidated during application restart
+up.jwt-secret=${UP_JWT_SECRET:}
+up.domain=${UP_DOMAIN:http://localhost:8080}
+up.dev.cors=${UP_DEV_CORS:false}
```

Comment:
profile-specific .properties file should only overwrite relevant properties, the rest is inherited from main application.properties